lib/shared/community_icon.dart

```diff
@@ -26,8 +26,10 @@ class CommunityIcon extends StatelessWidget {
               )
             : null);
 
+    if (community?.icon == null || community!.icon!.isEmpty) return placeholderIcon;
```

Comment:
Could this be simplified as 

```
if (community?.icon?.isNotEmpty != true)
```
?

Reply:
Addressed in 819d861d04d66f2f1699d94c1a905cddeb931c38